### PR TITLE
Added PT-BR version to readme file

### PR DESCRIPTION
**Pull request by mafemello**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -118,3 +122,115 @@ docs/CONTRIBUTING.md
...
+
+para impulsionar o script de configuração de compilação `configure`. Para isso, você
+precisa ter GNU Build System (Autotools) instalado. Plataformas baseadas em debian/ubuntu, 
+você pode instalar os softwares necessários com:
+
+```
+$ sudo apt install automake autoconf libtool
+```
+
+Por outro lado, se você obteu o software de __distribution repository__, geralmente como tarball,
```
Comment (PauloMRocha) [inline README.md:151]: I think that "se você obteve" would be better here

File: README.md [hunk 2]
```diff
@@ -118,3 +122,115 @@ docs/CONTRIBUTING.md
...
+
+## O Jogo
+ 
+O jogo se passa em uma área retangular onde uma cobra continuamente
+se move em uma das quatro direções: esquerda, direita, para cima e para baixo --- ela nunca
+para de se mover. Conforme a cobra se move ela perde energia e, se toda sua energia se esgotar, a
+cobra morre. Para recuperar energia, a cobra precisa comer pedaços de comida que
+são constantemente substituídos em posições aleatórias.
+
+Tenha cuidado, no entanto. As bordas da arena são eletrificadas e matariam a cobrinha se forem tocadas. Além disso, lembre-se que a cobra é venenosa e também morreria se
```
Comment (PauloMRocha) [inline README.md:214]: The verbal time is wrong here.
I sugest to use the sentences : "As bordas da arena são eletrificadas e matariam a cobrinha se fossem tocadas" and "Além disso, lembre-se que a cobra é venenosa e também morreria caso se mordesse acidentalmente".

File: README.md [hunk 3]
```diff
@@ -118,3 +122,115 @@ docs/CONTRIBUTING.md
 
 which contains important information.
 
+
+
+# PT-BR
+## Introdução
+Snaskii é um jogo de cobra ASCII muito simples, uma homenagem ao clássico
```
Comment (PauloMRocha) [inline README.md:129]: I think that "jogo da cobrinha" is a more common name in Brazil, but "jogo de cobrea" is ok too.

**Review by PauloMRocha — CHANGES_REQUESTED** [review]

**Comment by SusyDutra** [comment]
Changed the suggestions @PauloMRocha made

**Review by joaomisson — APPROVED** [review]
LGTM

**Pull request closed without merging**